bubblewrap.c

```diff
@@ -2201,8 +2207,11 @@ main (int    argc,
   if (proc_fd != -1)
     close (proc_fd);
 
-  if (opt_sync_fd != -1)
-    close (opt_sync_fd);
+  if (!no_reaper)
```

Comment:
We don't need this hunk, since we can rely on `sync_fd = -1` in this case right?